### ISSUE TITLE
System.IO.FileNotFoundException: Please be sure a sound file exists at the specified location.

### ISSUE THREAD

**Issue by mousology** [RandomEngy/VidCoder#1006]
### What were you doing when VidCoder crashed?

encoding multiple video files for size reduction
already done that multiple times before without a problem
i cant find a problem with output files
only reporting because the error message said report, and i really appreciate your software

### Crash details

```text
VidCoder 7.12 Portable
System.IO.FileNotFoundException: Please be sure a sound file exists at the specified location.
File name: 'C:\Users\Mostafa\AppData\Local\Temp\RarSFX0\Encode_Complete.wav'
   at System.Media.SoundPlayer.ValidateSoundFile(String fileName)
   at System.Media.SoundPlayer.LoadAndPlay(Int32 flags)
   at VidCoder.Services.ProcessingService.PlayEncodeCompleteSound() in C:\git\VidCoder\VidCoder\Services\ProcessingService.cs:line 2488
   at VidCoder.Services.ProcessingService.<>c__DisplayClass218_0.<OnEncodeCompleted>b__0() in C:\git\VidCoder\VidCoder\Services\ProcessingService.cs:line 2401
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by RandomEngy** [comment]
Thanks for the report. It looks like somehow the "encode complete" sound wasn't there, and that caused it to crash. I'm not sure how this happened to you, but you can work around it by disabling "Play sound on queue completion". The next release will have a fix to make it stop crashing when that sound file is missing.

**Comment by RandomEngy** [comment]
Fixed in version 7.15.
